server/clip_server/executors/clip_torch.py

```diff
@@ -26,6 +26,23 @@ def __init__(
         minibatch_size: int = 16,
         **kwargs,
     ):
+        """Encode images and texts into embeddings where the input is an iterable of raw strings.
+
+        Each image and text must be represented as a string. The following strings are acceptable:
```

Comment:
It is not true. An image document containing `.tensor` is also accepted as the input.